Traceback (most recent call last):
  File "/Users/jinseopsong/coding/LD50-Prediction/src/run.py", line 94, in <module>
    t.train()
  File "/Users/jinseopsong/coding/LD50-Prediction/src/run.py", line 62, in train
    out = self.model(batch)
  File "/Users/jinseopsong/Library/Python/3.9/lib/python/site-packages/torch/nn/modules/module.py", line 1532, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
  File "/Users/jinseopsong/Library/Python/3.9/lib/python/site-packages/torch/nn/modules/module.py", line 1541, in _call_impl
    return forward_call(*args, **kwargs)
  File "/Users/jinseopsong/coding/LD50-Prediction/src/model.py", line 62, in forward
    x = self.conv1(x, edge_index).relu()
  File "/Users/jinseopsong/Library/Python/3.9/lib/python/site-packages/torch/nn/modules/module.py", line 1532, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
  File "/Users/jinseopsong/Library/Python/3.9/lib/python/site-packages/torch/nn/modules/module.py", line 1541, in _call_impl
    return forward_call(*args, **kwargs)
  File "/Users/jinseopsong/Library/Python/3.9/lib/python/site-packages/torch_geometric/nn/conv/gcn_conv.py", line 241, in forward
    edge_index, edge_weight = gcn_norm(  # yapf: disable
  File "/Users/jinseopsong/Library/Python/3.9/lib/python/site-packages/torch_geometric/nn/conv/gcn_conv.py", line 109, in gcn_norm
    deg_inv_sqrt = deg.pow_(-0.5)
RuntimeError: Found dtype Long but expected Float